src/deepsparse/clip/zeroshot_pipeline.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, List, Type
+
+import numpy as np
+from numpy import linalg as la
+from pydantic import BaseModel, Field
+
+from deepsparse.clip import CLIPTextInput, CLIPVisualInput
+from deepsparse.pipeline import BasePipeline, Pipeline
+from scipy.special import softmax
+
+
+__all__ = ["CLIPZeroShotInput", "CLIPZeroShotOutput", "CLIPZeroShotPipeline"]
+
+
+class CLIPZeroShotInput(BaseModel):
+    """
+    Input for the CLIP Zero Shot Model
+    """
+
+    image: CLIPVisualInput = Field(
+        description="Path to image to run zero-shot prediction on."
+    )
+    text: CLIPTextInput = Field(description="List of text to process")
+
+
+class CLIPZeroShotOutput(BaseModel):
+    """
+    Output for the CLIP Zero Shot Model
+    """
+
+    # TODO: Maybe change this to a dictionary where keys are text inputs
+    text_scores: List[Any] = Field(description="Probability of each text class")
+
+
+@BasePipeline.register(task="clip_zeroshot", default_model_path=None)
+class CLIPZeroShotPipeline(BasePipeline):
+    def __init__(self, visual_model_path: str, text_model_path: str, **kwargs):
+        """
+        Pipeline designed to run zero-shot classification given a list of images and
```

Comment:
As convention, we push the docstring for init function above init function definited (under `class CLIPZeroShotPipeline(...)`. Also, please add params to the docstring